3-enrich/stream-enrich/project/Dependencies.scala

```diff
@@ -42,7 +42,7 @@ object Dependencies {
     val json4s               = "3.2.11"
     val pureconfig           = "0.8.0"
     val snowplowRawEvent     = "0.1.0"
-    val snowplowCommonEnrich = "0.32.0"
+    val snowplowCommonEnrich = "0.34.0"
```

Comment:
I don't see this https://github.com/snowplow/snowplow/blob/master/3-enrich/stream-enrich/core/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/Enrich.scala#L216-L243 being modified.

Reply:
Thanks for spotting - those changes were apparently lost after rebasing, will fix. I'll also be doing some smoke testing today which should help prevent issues like these